### PR TITLE
Format main.star

### PR DESCRIPTION
**Pull request by digiter**

### CODE REVIEW

File: config/main.star [hunk 1]
```diff
@@ -19,136 +21,156 @@ FLUTTER_GIT = 'https://chromium.googlesource.com/external/github.com/flutter/flu
...
-  config_dir = '.',
-  tracked_files = [
-    'cr-buildbucket.cfg',
-    'luci-logdog.cfg',
-    'luci-milo.cfg',
-    'luci-notify.cfg',
-    'luci-scheduler.cfg',
-    'project.cfg',
-  ],
+    config_dir='.',
```
Comment (dnfield) [inline config/main.star]: This looks like it got four spaces instead of two, and it's not entirely clear why.

File: config/main.star [hunk 2]
```diff
@@ -19,136 +21,156 @@ FLUTTER_GIT = 'https://chromium.googlesource.com/external/github.com/flutter/flu
...
-luci.logdog(
-  gs_bucket = 'chromium-luci-logdog',
-)
+luci.logdog(gs_bucket='chromium-luci-logdog',)
 
 luci.milo(
-  logo = 'https://storage.googleapis.com/chrome-infra-public/logo/flutter-logo.svg',
-  favicon = 'https://storage.googleapis.com/flutter_infra/favicon.ico',
+    logo=
+    'https://storage.googleapis.com/chrome-infra-public/logo/flutter-logo.svg',
```
Comment (dnfield) [inline config/main.star]: These two lines are harder to read.

File: config/main.star (hunk 1 above)
Comment (dnfield) [reply config/main.star]: Hm.  It looks like yapf wants to make this like python, where using double indendtation for line breaking reasons is common.  It looks strange to me but ... *shrug*

**Comment by digiter** [comment]
Probably work on this later when I got time.

**Pull request closed without merging**